scripts/setup-services.sh

```diff
@@ -83,15 +87,15 @@ cd services
 echo "Setting up login.webmaker.org"
 cd login.webmaker.org
 cp env.sample .env
-sudo npm install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
+sudo yarn install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
```

Comment:
From what I read from here https://github.com/yarnpkg/yarn/issues/1961 it looks like it is